### ISSUE TITLE
Request:Trivia challenge

### ISSUE THREAD

**Issue by Gishi1** [justinxxvii/python-bing-bot#8]
bing has added a new Trivia challenge! i was able to solve it by a simple bing search. whats your take on this? and how easy should it be to implement to the bing bot?

**Comment by justinxxvii** [comment]
This is weird. Today's question is about the tallest building in the USA. I
went ahead and entered junk data so I could unlock the hint. Then I clicked
the hint link. It forced a search for "Tallest building in United States",
and there was an area under the search box with a link to "One World Trade
Center". It will take experimentation to see if the answer is always
displayed like this. But basically if the hint page is always like this it
would be as simple as putting junk data as the three guesses, clicking the
hint link, then running a query on the xpath to find the link that is
shown, and getting the text off of it.

Will have to keep watching it to see what it does.

**Comment by Gishi1** [comment]
yes i agree but entering in junk each time could get the account flagged. wouldn't it be better to search the question if it pops up when its searching? just add a quick check in each search to see if its on the page then search the question and enter it as a search...

**Comment by rockylikespocky** [comment]
I've heard quite a few people have had their accounts banned this new coming year so I wouldn't risk using junk answers. Maybe pull the question into a search, find the first link, change it into text form and input it as the answer. I wouldn't want to risk all my accounts for 1 extra point a day.

**Comment by Gishi1** [comment]
now they are calling it "Trivia Tuesday" any progress on this?

**Comment by rockylikespocky** [comment]
This week they give you 3 different questions and Bing won't let me accumulate Desktop points unless you answer the questions first. Anyone else getting this? Mobile works fine.

**Comment by Gishi1** [comment]
ya i had to do mine first as well.
